default_app/default_app.js

```diff
@@ -15,7 +15,10 @@ exports.load = (appUrl) => {
       height: 600,
       autoHideMenuBar: true,
       backgroundColor: '#FFFFFF',
-      useContentSize: true
+      useContentSize: true,
+      webPreferences: {
+        preload: path.join(__dirname, 'preload.js')
```

Comment:
Curious as to why you are using a preload script here instead of just putting it directly in `index.html`?